doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -203,17 +203,35 @@ The [Cinder Configuration]( https://docs.openstack.org/cinder/latest/configurati
 
 [Ceph](https://ceph.io/) is the default CNTT Reference Architecture storage backend and is discussed below.
 
-#### 4.2.4.1. Ceph
+#### 4.2.4.1. Ceph Storage Cluster
+The Ceph storage cluster is deployed on bare metal hardware. The minimal configuration is a cluster of three bare metal servers to ensure High availability. The Ceph Storage cluster consists of 3 main daemon types:
+-	Monitor
+-	OSD (object storage daemon)
+-	RadosGW (Rados Gateway)
 
-**Content to be developed**
--	Types of storage (incl NVMe)
--	Ceph
-    -	Ceph Configuration
-    -	Cinder Gateway, Swift Gateway
--	How many nodes to meet SLA
--	Sizing rules (IOPS, SSD vs. SATA, NICs, etc).
+Ceph monitors maintain a master copy of the maps of the cluster state required by Ceph daemons to coordinate with each other. Ceph OSD handle the data storage (read/write data on the physical disks), data replication, recovery, rebalancing, and provides some monitoring information to Ceph Monitors. The RadosGW provides Object Storage RESTful gateway with a Swift-compatible API for Object Storage.
+
+<p align="center"><img src="../figures/Figure_4_x_Ceph.png" alt="Ceph Storage System"></br>Figure 4-2. Ceph Storage System.</p>
+
+**BIOS Requirement for Ceph servers**
 
+| BIOS/boot Parameter | Control Srever |
+|-------------|----------------|
+| Boot disks | RAID 1 |
 
+How many nodes to meet SLA :
+-	minimum: three bare metal servers where Monitors are collocated with OSD. Note: at least 3 Monitors and 3 OSDs are reuired for High AVailability. 
+
+HW specifications :
+-	the boot disks are SSD disks with a minimum capacity of 120GB
+-	for an IOPS oriented cluster (full SSD),  the journal can be hosted on OSD disks
+-	for a capacity oriented cluster (HDD), the journal must be hosted on dedicated SSD disks
+
+Sizing rules :
+-	Minimum of 6 disks per server
+-	Replication factor : 3
+-	1 Core-GHz per OSD
+-	16GB RAM baseline + 2-3 GB per OSD
 
```

Comment:
Why we need 6 Disk , it is based on IOPS, no need to mention number of disks

Reply:
@ASawwaf the disk drives requirements are also driven by practical considerations to minimize contentions such as not running monitors and OSDs on the same disk or multiple OSDs on the same disk. As you know similar restrictions exist for Journals. @samuelhellec can justify why six but given just the # of OSDs and monitors I can see why.

Reply:
agree to remove it it might not be relevant for the RA

Reply:
@ASawwaf I don't think we need to remove. Maybe add a justification. Do you agree?